docs/install_guides/installing-openvino-docker-windows.md

```diff
@@ -1,120 +1,113 @@
+.. tab:: Host Operating Systems
 
-You can use [available Dockerfiles](https://github.com/openvinotoolkit/docker_ci/tree/master/dockerfiles) or generate a Dockerfile with your setting via [DockerHub CI Framework](https://github.com/openvinotoolkit/docker_ci).
-The Framework can generate a Dockerfile, build, test, and deploy an image with the Intel® Distribution of OpenVINO™ toolkit. You can find device specific steps to configure an Intel® Distribution of OpenVINO™ toolkit Dockerfile below.
+  * Windows 10, 64-bit Pro, Enterprise or Education (1607 Anniversary Update, Build 14393 or later) editions
+  * Windows Server 2016 or higher
+  
+.. tab:: Additional Requirements for GPU
 
-## Configure and Run the Docker* Image for CPU
+  GPU Acceleration in Windows containers feature requires to meet Windows host, OpenVINO and Docker requirements:
 
-## Install Additional Dependencies
+  - [Windows requirements](https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/gpu-acceleration):
+    - The container host must be running Windows Server 2019 or Windows 10 of version 1809 or higher.
+    - The container base image must be `mcr.microsoft.com/windows:1809` or higher. Windows Server Core and Nano Server container images are not currently supported.
+    - The container host must be running Docker Engine 19.03 or higher.
+    - The container host must have GPU running display drivers of version WDDM 2.5 or higher.
+  - [GPU requirement for OpenVINO](https://docs.openvino.ai/latest/openvino_docs_install_guides_installing_openvino_windows.html#Install-GPU): Intel Graphics Driver for Windows of version 15.65 or higher.
+  - [Docker isolation mode requirement](https://docs.microsoft.com/en-us/virtualization/windowscontainers/manage-containers/hyperv-container):
+    - Windows host and container version tags must match.
+    - [Windows host and container isolation process support](https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/version-compatibility)
 
-### Install CMake
+@endsphinxdirective
 
-To add CMake to the image, add the following commands to the Dockerfile:
 
-```bat
-RUN powershell.exe -Command `
-    Invoke-WebRequest -URI https://cmake.org/files/v3.14/cmake-3.14.7-win64-x64.msi -OutFile %TMP%\\cmake-3.14.7-win64-x64.msi ; `
-    Start-Process %TMP%\\cmake-3.14.7-win64-x64.msi -ArgumentList '/quiet /norestart' -Wait ; `
-    Remove-Item %TMP%\\cmake-3.14.7-win64-x64.msi -Force
-
-RUN SETX /M PATH "C:\Program Files\CMake\Bin;%PATH%"
-```
+## Installation Flow
 
-In case of proxy issues, please add the `ARG HTTPS_PROXY` and `-Proxy %%HTTPS_PROXY%` settings to the `powershell.exe` command to the Dockerfile. Then build a Docker image:
+There are two ways to install OpenVINO with Docker. You can choose either of them according to your needs:
+* Use a prebuilt image. Do the following steps:
+  1. Get a prebuilt image from [Docker Hub](https://hub.docker.com/u/openvino).
```

Comment:
Can you please add the link to the Azure Marketplace here? We are recently added this distribution channel.

https://azuremarketplace.microsoft.com/en-us/marketplace/apps/intel_corporation.openvino

Reply:
OK, updated.